inlong-manager/manager-dao/src/main/java/org/apache/inlong/manager/dao/entity/InlongGroupEntity.java

```diff
@@ -35,6 +35,7 @@ public class InlongGroupEntity implements Serializable {
     private String description;
     private String mqType;
     private String mqResource;
+    private String backupMqResource;
```

Comment:
It is not recommended to add special backup configurations.

You can put them in the ext_params field, or put them in the inlong_group_ext table.